### PR TITLE
Decapode + udata-datalake-service = Hydra

### PR DESCRIPTION
**Pull request by sixtedemaupeou**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,26 +1,61 @@
...
 [project.urls]
 Home = "https://github.com/opendatateam/udata-datalake-service"
 
 [tool.flit.module]
-name = "udata_datalake_service"
+name = "hydra"
 
 [project.scripts]
- udata-datalake-service = "udata_datalake_service.cli:cli"
+hydra = "hydra.cli:run"
```
Comment (maudetes) [inline pyproject.toml:60]: We need to complete with correct cli entries

**Review by maudetes — COMMENTED** [review]
I added the minimal stuff for a working CI, 

I replaced the setup.cfg + requirements files with a pyproject.toml config that we can update.

They were some tests failing, in particular on the crawler side due to `HEAD` call being replaced by `GET`. Not sure they are all relevant.
At the moment they don't check for produce signal being sent.

**Review by maudetes — COMMENTED** [review]
A batch of review, I think we're almost good but I did not read all the code entirely

File: hydra/datalake_service.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                    KAFKA_URI,
+                    get_topic("resource.stored"),
+                    "datalake",
+                    resource_id,
+                    {
+                        "location": storage_location,
+                        "mime_type": mime_type,
+                        "filesize": filesize,
+                    },
+                    meta={"dataset_id": dataset_id},
```
[the remark this answers is not in the document]
Comment (sixtedemaupeou) [reply hydra/datalake_service.py:98]: Did we say we also wanted to include the message_type? Is there any point doing so as it is always the same as the topic?

File: hydra/kafka/consumer.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+async def process_message(key: str, message: dict, topic: str) -> None:
+    log.info("Received message")
+    dataset_id = message["meta"]["dataset_id"]
+
+    pool = await context.pool()
+    async with pool.acquire() as connection:
+        if topic == get_topic("resource.created"):
+            resource = message["value"]["resource"]
```
[the remark this answers is not in the document]
Comment (sixtedemaupeou) [reply hydra/kafka/consumer.py:17]: Are we going to remove the `resource` level in the message? If so we should also update https://github.com/geoffreyaldebert/udata-kafka-event-testing/blob/e526427d8be2a01d2f10b03201e2b533d405e713/messages.json#L9

**Pull request merged**